src/index.js

```diff
@@ -20,12 +20,33 @@ function deleteSingleTask() {
   });
 }
 
+function clear() {
+  const clear = document.getElementById('clear-button');
+  clear.addEventListener('click', () => {
+    const checkboxes = document.getElementsByClassName('task');
+    const checkboxesArr = Array.from(checkboxes);
+
+    let deleteList = [];
+
+    checkboxesArr.forEach((el, i) => {
+      if (el.classList.contains('task-done')) deleteList.push(i);
+    });
```

Comment:
- [ ] Hi, you did not implement the filter function here. It would be nice to meet specific project requirements because, in the real world, you might be given such tasks and based to implement them using the company's standard code. 

_**Hint**_: What you want to do is to filter through the array and return those tasks that do not have their `completed` key set to `true`. Kindly refactor the code to use `array.filter` as it is a requirement for this milestone 👍 

### You can reach out to **_RoyM_** on slack and schedule a call with him. He is a senior code reviewer and he has asked you to schedule a call with him. 🙏

Reply:
Thanks for your feedback, but I mush inform you that I did: https://github.com/Kakalanp/TO-DO-list/blob/c712489940d80e8d01b2587ed59b3e9931671d72/src/modules/deleteTask.js#L9
It's not the same way you're suggesting, but I did implement the changes.